sealtk/noaa/gui/Window.cpp

```diff
@@ -191,6 +204,16 @@ Window::Window(QWidget* parent)
                 t.value<kwiver::vital::timestamp::time_t>());
             }
           });
+  connect(d->ui.tracks->selectionModel(),
+          &QItemSelectionModel::selectionChanged,
+          this, [d](QItemSelection const& selection){
+            d->updateTrackSelection(selection);
+          });
+  connect(d->ui.actionCreateDetection, &QAction::triggered,
+          this, [d]{
+            d->eoWindow.player->setActiveTool(d->eoWindow.createTrackTool);
```

Comment:
Still want to test this, but it looks like the tool just won't do anything, which is okay. (Especially since, as set up, we want this to work across all modalities without necessarily having imagery loaded in all of them.)